blocks/express_form/templates/hosting_two_column.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+defined('C5_EXECUTE') or die('Access Denied.');
+
+/** @var \Concrete\Core\Block\View\BlockView $view */
+/** @var \Concrete\Core\Express\Form\Renderer|null $renderer */
+/** @var string|null $success */
+/** @var string $bID */
+/** @var \Concrete\Core\Error\ErrorList\ErrorList|null $error */
+/** @var \Concrete\Core\Captcha\CaptchaInterface|null $captcha */
+/** @var string $displayCaptcha "0" or "1" */
+/** @var string $submitLabel */
+?>
+<div class="ccm-block-express-form hosting-inquiry-form">
+    <?php if (isset($renderer)) {
+    ?>
+        <div class="ccm-form">
+            <a name="form<?=$bID; ?>"></a>
+
+            <?php if (isset($success)) {
+        ?>
```

Comment:
I see the core has this same syntax messiness. If it's not too much work, having your editor reformat files like this to match our code style guidelines would be much appreciated when adding code to projects.